docs/sql-ref-syntax-aux-conf-mgmt-reset.md

```diff
@@ -19,4 +19,20 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Reset all SQLConf properties to the default values. After RESET command, executing SET command will output empty.
```

Comment:
Can we say "Reset all the properties specific to the current session to their default values" ? I would like to remove SQLConf as even non sql properties are being cleared.